src/main/java/net/dv8tion/jda/internal/handle/GuildUpdateHandler.java

```diff
@@ -99,7 +99,7 @@ protected Long handleInternally(DataObject content)
         {
             guild.setBannerId(bannerId);
         }
-        if (!Objects.equals(vanityCode, guild.getVanityUrl()))
+        if (!Objects.equals(vanityCode, guild.getVanityCode()))
```

Comment:
We need to add events for all of these new properties.

Reply:
I already did.

Reply:
Ah, I was looking at an older version of the changes.